src/main/java/org/jalasoft/moi/controller/services/FileService.java

```diff
@@ -35,10 +35,10 @@
      *
      * @return a parameters object
      */
-    public Parameters saveFileB64(String name, String codeB64, String filePath, String extension, Language language) throws IOException {
+    public Parameters saveFileB64(String name, String codeB64, String filePath, Language language) throws IOException {
```

Comment:
We need to test if it is still needed to have three different 'saveFIle' methods.